ui/intermittents-view/helpers.js

```diff
@@ -0,0 +1,100 @@
+import moment from "moment";
+import { bugzillaDomain } from "./constants";
+
+//be sure to wrap date arg in a moment()
+export const ISODate = date => date.format("YYYY-MM-DD");
```

Comment:
I've started following this convention in the Airbnb style guide, especially in the "helper" functions.  I think we should follow it here, too: https://github.com/airbnb/javascript#functions

Reply:
@camd Are we not using ES6 fat arrow functions anymore? I'm curious as to why they aren't in the Airbnb style guide since they are function expressions.

Reply:
I think part of the idea is so that stack traces are easier to follow.  I haven't observed this myself, I must admit.  But I think this makes the function names show up in the stack traces.